### ISSUE TITLE
[question] Replace integer by int in php @param 

### ISSUE THREAD

**Issue by flashios09** [20Tauri/DoxyDoxygen#71]
hi @20Tauri ,

First of all thanks for this awesome plugin.
When i have a php method like this:
```php
public function test(int $value)
{
    // ...
}
```
and i generate the doc block, it give me this result:
```php
/**
 * { function_description }
 *
 * @param integer $value The value
 */
public function test(int $value)
{
    // ...
}
```
I have a problem with **PHP CODE SNIFFER**
> Expected  "int" but found "integer" for parameter type

Is it possible to replace `@param integer $value The value` by `@param int $value The value` ?

**Comment by 20Tauri** [comment]
Hi,

Thank you for this suggestion

A new option `phpdoc_short_primitives` has been created to handle this case.
Both "Integer" and "Boolean" will be shorten

This change is included in version 0.56.0 (currently available on "evolution" channel)

Regards,

**Comment by flashios09** [comment]
@20Tauri thanks, i installed the 0.56.0(manually), set `"phpdoc_short_primitives": true` and it works now ;)
